src/main/native/cpp/UsbCameraImpl.h

```diff
@@ -1,30 +1,33 @@
 /*----------------------------------------------------------------------------*/
-/* Copyright (c) FIRST 2016. All Rights Reserved.                             */
+/* Copyright (c) 2016-2017 FIRST. All Rights Reserved.                        */
 /* Open Source Software - may be modified and shared by FRC teams. The code   */
 /* must be accompanied by the FIRST BSD license file in the root directory of */
 /* the project.                                                               */
 /*----------------------------------------------------------------------------*/
 
-#ifndef CS_USBCAMERAIMPL_H_
-#define CS_USBCAMERAIMPL_H_
+#ifndef SRC_USBCAMERAIMPL_H_
+#define SRC_USBCAMERAIMPL_H_
 
 #include <atomic>
+#include <memory>
+#include <string>
 #include <thread>
+#include <utility>
 #include <vector>
 
-#ifdef __linux__
-#include <linux/videodev2.h>
-#endif
-
-#include "llvm/raw_ostream.h"
-#include "llvm/SmallVector.h"
-#include "llvm/STLExtras.h"
-#include "support/raw_istream.h"
+#include <llvm/STLExtras.h>
+#include <llvm/SmallVector.h>
+#include <llvm/raw_ostream.h>
+#include <support/raw_istream.h>
 
 #include "SourceImpl.h"
 #include "UsbCameraBuffer.h"
 #include "UsbCameraProperty.h"
 
+#ifdef __linux__
+#include <linux/videodev2.h>
+#endif
+
```

Comment:
This system include got moved below local includes (presumably because of the ifdef?).